lib/build.gradle

```diff
@@ -50,6 +50,8 @@ cargo {
     // Set to "debug" to enable logging. Or "release" to make a production build.
     // https://github.com/mozilla/rust-android-gradle/issues/38
     profile = gradle.startParameter.taskNames.any { it.toLowerCase().contains("debug") } ? "debug" : "release"
+    pythonCommand = "/opt/homebrew/bin/python3"
```

Comment:
Use `python3` instead of `python` to fix below error:

![Uploading Screenshot 2023-10-31 at 21.47.28.png…]()